_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,466 @@
+      </tr>
+      <tr>
+        <td>SSH</td>
+        <td>TCP</td>
+        <td>22</td>
+        <td>My IP</td>
+        <td>optional</td>
+      </tr>
+      <tr>
+        <td>HTTP</td>
+        <td>TCP</td>
+        <td>80</td>
+        <td>Anywhere</td>
+        <td>optional</td>
+      </tr>
+      <tr>
+        <td>HTTPS</td>
+        <td>TCP</td>
+        <td>443</td>
+        <td>Anywhere</td>
+        <td>optional</td>
+      </tr>
+    </table>
+
+13. Choose **Review and Launch**.
+
+    The system may warn you that the selection is not free tier eligible. You can ignore this for now.
+
+14. Press **Launch**.
+
+    The system prompts you for a key pair.
+
+15. Select **Create a new keypair** or **Choose an existing key pair**.
+16. Select **Launch Instances**.
+
+    The system launches your instance.
+
+    ![](/storage/images/aws-launch-status.png)    
+
+During the instance launch, the machine image on the instance performs some initial setup processes. These processes take a few minutes depending on the minutes, typically thought it should not take more than 5 mins. While this is happening your **Status Checks** reflect the **Initializing** status.
+
+
+## Task 2: Test your Gaia server
+
+Now, you are ready to test your Gaia server and make sure it is up and running.
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Choose **All services > EC2**.
+
+   The system displays the **EC2 Dashboard**.
+
+   ![](/storage/images/ec2-dashboard.png)    
+
+3. Select **Running Instances**.
+
+   The system displays your running instances.
+
+4. Locate your recently launched Gaia server.
+
+   Make sure the instance shows as running and **Status Checks** are complete.
+   Completed status checks ensures the Gaia processes and service were started.
+
+5. Select the **Description** tab.
+
+   ![](/storage/images/ec2-instance.png)   
+
+6. Locate the **IPv4 Public IP** value.
+7. Copy the IP and paste it in your browser.
+
+   You should see a message that your connection is not private.
+
+8. Press **Advanced**.
+
+   ![Hub test](/storage/images/private-connection.png)
+
+9. Choose to proceed.
+10. Extend the IP with the `PUBLIC_IP/hub_info` tag like so.
+
+    You should see a response from your Gaia hub!
+
+    ![Hub test](/storage/images/aws-hub.png)
+
+    At this point, you should see a **Not secure** message in the browser. That's because you haven't yet enabled SSL certification.
+
+
+## Task 3: Configure a domain name
+
+At this point, you can point a domain to your Gaia hub. Although it's not required, it is highly recommended. If you use a domain, you can migrate your instance to a different server (or even provider such as Azure or Dropbox) at any time, and still access it through the domain URL. Just point your domain at the IP address for your EC2 instance. Use an `A Record` DNS type.
```

Comment:
It may be worth mentioning that while HTTPS may not required to simply have a hub running via HTTP, it _is_ required to connect the hub to your Blockstack ID for usage. 

i.e. the Blockstack UI will reject any non-HTTPs URLs that users attempt to set for configuration

Reply:
Did this.